recipes-graphics/spark/spark_git.bb

```diff
@@ -0,0 +1,100 @@
+include pxcore.inc
+
+DEPENDS_append = " curl freetype util-linux libjpeg-turbo libpng pxcore-libnode giflib sqlite3 wpeframework-tools-native wpeframework wpeframework-clientlibraries"
+
+SRC_URI += "file://Spark.pc \
+    file://0001-nanosvg-patches.patch \
+    file://0002-pxScene-disable-2DMultisampleEXT.patch \
+    file://0003-spark-wpeframework-compositor.patch \
+    file://0004-dukluv-git.patch \
+    file://0005-pxScene-essos-support-for-shared-lib.patch \
+    file://0006-pxScene-initApp-load-issue-fix.patch \
+    file://0007-node-v10.15.3_mods.patch \
+"
+
+inherit cmake pkgconfig
+
+PACKAGECONFIG ?= " \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'wayland westeros', 'wpeframework', d)} \
+"
+
+PACKAGECONFIG[rtremote]     = "-DBUILD_RTCORE_LIBS=ON -DBUILD_RTCORE_STATIC_LIB=OFF,,,rtcore rtremote"
+PACKAGECONFIG[wayland]      = "-DBUILD_WITH_WAYLAND=ON -DPXCORE_WAYLAND_EGL=ON -DBUILD_PXSCENE_WAYLAND_EGL=ON,,wayland"
+PACKAGECONFIG[westeros]     = "-DBUILD_WITH_WESTEROS=ON -DPXCORE_WAYLAND_EGL=ON -DBUILD_PXSCENE_WAYLAND_EGL=ON,,westeros"
+PACKAGECONFIG[wpeframework] = "-DBUILD_WITH_WPEFRAMEWORK=ON -DPXCORE_WPEFRAMEWORK=ON,,wpeframework"
+
+COMPOSITOR          ?= "${@bb.utils.contains('PACKAGECONFIG', 'wpeframework', 'wpeframework', 'wayland_egl', d)}"
+LIBRTCORE_SUBDIR    ?= "${@bb.utils.contains('PACKAGECONFIG', 'wpeframework', 'wpe', 'egl', d)}"
+
+EXTRA_OECMAKE += " \
+    -DBUILD_WITH_TEXTURE_USAGE_MONITORING=ON \
+    -DPXCORE_COMPILE_WARNINGS_AS_ERRORS=OFF \
+    -DPXSCENE_COMPILE_WARNINGS_AS_ERRORS=OFF \
+    -DCMAKE_SKIP_RPATH=ON \
+    -DPXCORE_MATRIX_HELPERS=OFF \
+    -DBUILD_PXWAYLAND_SHARED_LIB=OFF \
+    -DBUILD_PXWAYLAND_STATIC_LIB=OFF \
+    -DPREFER_SYSTEM_LIBRARIES=ON \
+    -DDISABLE_TURBO_JPEG=ON \
```

Comment:
Can you please make a turbo jpeg a packageconfig that is enabled by default, but includes the dependency over the turbo_jpeg?